consensus/istanbul/config.go

```diff
@@ -28,11 +28,13 @@ type Config struct {
 	BlockPeriod    uint64         `toml:",omitempty"` // Default minimum difference between two consecutive block's timestamps in second
 	ProposerPolicy ProposerPolicy `toml:",omitempty"` // The policy for proposer selection
 	Epoch          uint64         `toml:",omitempty"` // The number of blocks after which to checkpoint and reset the pending votes
+	QuorumFormula   uint64         `toml:",omitempty"` // The quorum formula used, 0 ==> 2f+1, 1 ==> Ceil(2N/3), 2 ==> N-f
 }
 
 var DefaultConfig = &Config{
 	RequestTimeout: 10000,
 	BlockPeriod:    1,
 	ProposerPolicy: RoundRobin,
 	Epoch:          30000,
+	QuorumFormula:    1,
```

Comment:
Should the default be `0` (existing behavior `2F+1`)? I recall you mentioned the need for adding this switch to be so that you can update it on the fly to test that the fix works.

Reply:
thanks for the confirmation @panghalamit . For clarity, we will use the value set to `1` (`ceil(2N/3)` for dev/it) and set it to `0` before pushing to prod

Reply:
let's we utilize the existing switches in dev/it so clustermanager can conditionally provision an environment variable in the quorum container, and eth-boot will then stick in the necessary switch to flip the default. this mechanism will be useful for us to test further enhancements with this formula in other IBFT locations.

as such, the default should be changed to `0` so it'll be the same as before in PROD if no additional formula switch is in place. we should propose a fix to Quorum once we are confident enough about this new formula. Then for PROD we want to pick up the fix via the official Quorum release once our proposal is merged.